docs/source/entities/quests.md

```diff
@@ -1 +1,20 @@
-# Quests
\ No newline at end of file
+# Quests
+
+Quests are a place to regroup all elements for a part of scenario or a scene.
+
+## Features
+Quests specific features are:
+* can be nested with "parent quest" field
+* a "date" field for reporting which will be displayed in side bar below pinned attributes
```

Comment:
You can replace "side bar" with `[profile sidebar](/features/profile-sidebar)` to keep it consistent with the rest of the docs